### ISSUE TITLE
Saltvannsøkologi + Marinøkologi

### ISSUE THREAD

**Issue by Caroline-A** [realfagstermer/realfagstermer#1389]
![image](https://user-images.githubusercontent.com/42732381/45552951-c5ba5a00-b832-11e8-9a66-dc0789644d58.png)
![image](https://user-images.githubusercontent.com/42732381/45552976-d8cd2a00-b832-11e8-8821-7dcd156bf2e4.png)

Foreslår sammenslåing, med marinøkologi som hovedtermen. Saltvannsøkologi har bare 13 dok.

**Comment by BioHeidi** [comment]
Ja, godt sett! Saltvannsøkologi er det vel ikke så mange som sier (lenger).

**Comment by Hypsibius** [comment]
Absolutt!
